projects/bootloader/inc/stm32f0xx/bootloader_mcu.h

```diff
@@ -23,3 +25,5 @@ extern uint32_t _vector_table_size;
 #define BOOTLOADER_RAM_START ((void *)&_ram_start)
 #define BOOTLOADER_RAM_SIZE ((size_t)&_ram_size)
 #define BOOTLOADER_VECTOR_TABLE_SIZE ((size_t)&_vector_table_size)
+#define BOOTLOADER_DEFAULT_LOCATION ((void *)&_bootloader_start)
```

Comment:
I think `BOOTLOADER_START` might be a clearer name for this, especially since we'll likely never have the bootloader at a different location (so not "default").